elasticdl/python/worker/worker.py

```diff
@@ -341,46 +340,44 @@ def get_model(self):
                     raise RuntimeError(
                         "PS pod %d cannot be initialized" % ps_id
                     )
-                for k, v in ps_params.items():
-                    self._non_embed_vars[k].assign(v)
+                dense_params.update(ps_params)
```

Comment:
We can move out the pull_dense_parameters  from the for loop, and use one call instead:

```
ps_params, uninit = self._ps_client.pull_dense_parameters(
     uninit_ps, self._model_versions_from_ps)
dense_params.update(ps_params)
```

Reply:
Good idea, fix it.